documentation/definition-of-done.md

```diff
@@ -1,20 +1,16 @@
-# Definition of Done:
+# Definition of Done
 
-- Step by step testing instructions
-- At least one peer review acceptance
-- Chromatic testing by design
-- Definition of done checklist from issue template (eventually)
-- If the route to a live page has been changed, a Permanent Redirect (301) has also been added to the nginx conf file
-- Build pipeline successful, including:
-  - JavaScript linting
+- Step-by-step testing instructions
+- At least one peer review approval
+- If the route to a live page has been changed, a Permanent Redirect (301) has also been added to the `nginx` conf files
+- Build pipeline successful, including passing tests for:
```

Comment:
Is there value in listing our CI steps here?  They run automatically - you don't need a checklist.

Reply:
Agree! I thought the same thing but didn't want to go too wild here, but I will!